examples/Synapse/Synapse.Tests.ps1

```diff
@@ -1,62 +1,80 @@
+  It 'Should contain a Synapse Workspace with the given name - Confirm-AzBPResource' {
+    #arrange
+    $params = @{
+      ResourceType = "SynapseWorkspace"
+      ResourceGroupName = $rgName
+      ResourceName = $workSpaceName
+    }
+
+    #act
+    $result = Confirm-AzBPResource @params
+
+    #assert
+    $result.Success | Should -Be $true
+  }
+
+  It "Should contain a Synapse Workspace named $workSpaceName - ConfirmAzBPResource" {
     #arrange
-    $rgName = 'rg-test'
-    $workspaceName = 'samplesynws'
+    $params = @{
+      ResourceType = "SynapseWorkspace"
+      ResourceGroupName = $rgName
+      ResourceName = $workSpaceName
+      PropertyKey = 'Name'
+      PropertyValue = $workSpaceName
+    }
 
+    #act
+    $result = Confirm-AzBPResource @params
+
+    #assert
+    $result.Success | Should -Be $true
+  }
+
+  It "Should contain a synapse workspace named $workSpaceName" {
     #act
     $result = Confirm-AzBPSynapseWorkspace -ResourceGroupName $rgName -WorkspaceName $workspaceName
 
     #assert
     $result.Success | Should -Be $true
   }
 
-  it 'Should contain a synapse workspace named samplesynws' {
+  It "Should contain a synapse workspace named $workSpaceName" {
     # Using custom assertion to check if the workspace is deployed
-    $rgName = 'rg-test'
-    $workspaceName = 'samplesynws'
-
     #act
     $result = Confirm-AzBPSynapseWorkspace -ResourceGroupName $rgName -WorkspaceName $workspaceName
 
     #assert
     $result | Should -BeDeployed
   }
 
-  it 'Should contain a synapse workspace named samplesynws in westus3' {
+  It "Should contain a synapse workspace named $workSpaceName in $location" {
     # Using custom assertion to check if the workspace is in the correct location
-    $rgName = 'rg-test'
-    $workspaceName = 'samplesynws'
-    $location = 'westus3'
-
     #act
     $result = Confirm-AzBPSynapseWorkspace -ResourceGroupName $rgName -WorkspaceName $workspaceName
 
     #assert
     $result | Should -BeInLocation $location
   }
 
-  it 'Should contain a synapse workspace named samplesynws in rg-test' {
+  It "Should contain a synapse workspace named $workSpaceName in $rgName" {
     # Using custom assertion to check if the workspace is in the correct resource group
-    $rgName = 'rg-test'
-    $workspaceName = 'samplesynws'
-
     #act
     $result = Confirm-AzBPSynapseWorkspace -ResourceGroupName $rgName -WorkspaceName $workspaceName
 
     #assert
-    $result | Should -BeInResourceGroup 'rg-test'
+    $result | Should -BeInResourceGroup $rgName
   }
 
-  it 'Should not contain a synapse workspace named nosamplesynws' {
-    #arrange
-    $rgName = 'rg-test'
-    $workspaceName = 'nosamplesynws'
-
+  It 'Should not contain a synapse workspace named nosamplesynws' {
     #act
     # The '-ErrorAction SilentlyContinue' command suppresses all errors.
     # In this test, it will suppress the error message when a resource cannot be found.
```

Comment:
This is missing the `no-<WorkspaceName>` parameter and switch

Reply:
Thanks! Fixed!